spec/controllers/bulk_task_assignments_controller_spec.rb

```diff
@@ -39,7 +39,7 @@
           get :create, params: { bulk_task_assignment: params }
           expect(response.status).to eq 200
           response_body = JSON.parse(response.body)
-          expect(response_body["data"].size).to eq 2
+          expect(response_body["queue_config"]["tabs"][1]["tasks"].count).to eq 2
```

Comment:
Make sure other tabs are also correct.
```suggestion
          expect(response_body["queue_config"]["tabs"][0]["tasks"].count).to eq 0
          expect(response_body["queue_config"]["tabs"][1]["tasks"].count).to eq 2
          expect(response_body["queue_config"]["tabs"][2]["tasks"].count).to eq 0
          expect(response_body["queue_config"]["tabs"][3]["tasks"].count).to eq 0
```